Assets/Python/RiseAndFall.py

```diff
@@ -523,14 +513,14 @@ def prepareChina(self):
 
 	def setupBirthTurnModifiers(self):
 		for iCiv in range(iNumPlayers):
-			if (iCiv >= iIndia and not gc.getPlayer(iCiv).isHuman()):
+			if iCiv >= iIndia and not gc.getPlayer(iCiv).isHuman():
 				data.players[iCiv].iBirthTurnModifier = gc.getGame().getSorenRandNum(11, "BirthTurnModifier") - 5 # -5 to +5
 		#now make sure that no civs spawn in the same turn and cause a double "new civ" popup
 		for iCiv in range(iNumPlayers):
-			if (iCiv > utils.getHumanID() and iCiv < iBrazil):
+			if iCiv > utils.getHumanID() and iCiv < iBrazil:

```

Comment:
`if iCiv > utils.getHumanID() and iCiv < iNumPlayers-1`

(Should be Canada by now already)